src/const.js

```diff
@@ -87,6 +93,22 @@ export const LOW_LEVEL_MODULES = [
   'system/events', 'sdk/system/events',
 ];
 
+export const DEPRECATED_ENTITIES = [
+  {
```

Comment:
Yeah - I looked into what we can change. Comma omission appears to only be allowed when the whole thing is on one line. So it doesn't seem like there's a great way to change it. We need a rule that says if you have `}];` you don't need `},];`